src/content/docs/new-relic-solutions/get-started/intro-new-relic2.mdx

```diff
@@ -0,0 +1,515 @@
+  
+  Learn more about network [here](/docs/network-performance-monitoring/get-started/npm-introduction/).
+
+  </Side>
+  <Side>
+
+<img
+  title="New Relic network"
+  alt="New Relic network"
+  src={NRNetworkMain}
+/>
+
+  </Side>
+  </SideBySide>
+
+# Mobile [#mobile]
+<SideBySide>
+  <Side>
+<img
+  title="New Relic mobile"
+  alt="New Relic mobile"
+  src={NRMobileMain}
+/>
+
+  </Side>
+  <Side>
+
+  The <InlinePopover type="mobile" /> capability sends telemetry directly from the end user's mobile app to New Relic. This allows us to analyze the app's performance on the user's device. This capability helps with the performance and crash troubleshooting of your Android, iOS, or hybrid mobile apps. Use mobile monitoring to improve your app's user experience. Or, study HTTP and network performance to work better with your backend teams.
+
+  Learn more about mobile [here](/docs/mobile-monitoring/new-relic-mobile/get-started/introduction-mobile-monitoring/).
+
+  </Side>
+  </SideBySide>
+
+# OpenTelemetry [#opentelemetry]
+<SideBySide>
+  <Side>
+
+  OpenTelemetry helps collect data from your applications and hosts, sending it to New Relic. New Relic's platform lets you analyze the data and fix any application problems. This toolkit simplifies the process of monitoring and troubleshooting your software. By utilizing OpenTelemetry, you can enhance the performance and reliability of your applications.
+
+  Learn more about OpenTelemetry [here](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-get-started-intro/).
+
+  </Side>
+  <Side>
+
+<img
+  title="New Relic OpenTelemetry"
+  alt="New Relic OpenTelemetry"
+  src={NROpenTelemetryMain}
+/>
+
+  </Side>
+  </SideBySide>
+
+# Serverless and Lambda [#serverless-lambda]
+<SideBySide>
+  <Side>
+
+<img
+  title="New Relic Lambda functions"
+  alt="New Relic Lambda functions"
+  src={NRLambdaMain}
+/>
+
+  </Side>
+  <Side>
+
+  We offer in-depth performance monitoring for your serverless AWS Lambda functions. Our Lambda layer gives one view into the details of your Lambda functions. You can understand what's happening in your serverless apps. Lambda monitoring is serverless and uses CloudWatch data and code-level tools to give you more in-depth data. 
+
+  Learn more about serverless and Lambda [here](/docs/serverless-function-monitoring/aws-lambda-monitoring/get-started/monitoring-aws-lambda-serverless-monitoring/).
+
+  </Side>
+  </SideBySide>
+
+# Service levels [#service-levels]
+<SideBySide>
+  <Side>
+
+  With New Relic, you can set and use service level indicators (SLIs) and objectives (SLOs) for your applications. Service levels measure the performance of a service from the end user (or client application) point of view. We measure service levels with indicators, comparing them to set objectives for performance and outcomes. 
```

Comment:
I think the buy in with service levels is that it gives you the ability to track improvements or degradations over time, then sets firm objectives that need to be "met" in the form of SLOs. But honestly this is such a nebulous and weird concept that I might ask Shawn/austin if its worth cutting since it's not even a capability, just something you can do with New Relic

Reply:
Shawn asked to include this one.